libs/newsletter-workflow/src/lib/steps/newsletterData/newsletterDesignLayout.ts

```diff
@@ -14,23 +14,18 @@ const markdownTemplate = `
 # Add the design brief and Figma design link for {{name}}
 
 Please share the following for **{{name}}**:
-- the design brief Google document
-- the signed off Figma design URL
-
-Does the Figma design file include images designs for thrashers?
-
-If so, please ensure the thrasher images are provided for both desktop and mobile widths.
-
+- The design brief Google document URL
+- The signed off and final Figma design URL, including thrasher designs for mobile and desktop
```

Comment:
Should this say "where required"?  Not all newsletters will necessarily have thrashers